### ISSUE TITLE
Ubuntu (.deb) the package is of bad quality

### ISSUE THREAD

**Issue by radist** [fathomssen/redtimer#81]
```
Lintian check results for /home/../../redtimer-v1.2-pre19-linux-x86_64.deb:
E: redtimer: maintainer-name-missing <travis@testing-gce-8b7f5c24-e40a-4703-b96e-c4ee88866a8a>
E: redtimer: maintainer-address-malformed <travis@testing-gce-8b7f5c24-e40a-4703-b96e-c4ee88866a8a>

```

**Comment by fathomssen** [comment]
Probably fixed with commit 7a81cc0

**Comment by fathomssen** [comment]
Please check again with https://github.com/fathomssen/redtimer/releases/tag/v1.2-pre20

**Comment by fathomssen** [comment]
Most things are fixed now with commit 85843cb:

```
$ lintian redtimer.deb 
E: redtimer: embedded-library opt/redtimer/lib/libfreetype.so.6: freetype
E: redtimer: embedded-library opt/redtimer/lib/libpng12.so.0: libpng
W: redtimer: missing-depends-line
E: redtimer: dir-or-file-in-opt opt/redtimer/
E: redtimer: dir-or-file-in-opt opt/redtimer/AppRun
E: redtimer: dir-or-file-in-opt opt/redtimer/lib/
E: redtimer: dir-or-file-in-opt ... use --no-tag-display-limit to see all (or pipe to a file/program)
W: redtimer: binary-without-manpage usr/bin/redtimer
W: redtimer: binary-without-manpage usr/bin/redtimercli
```

I will leave it so far for now.

If you have any suggestions as how to fix the above errors and warnings, I will be glad to hear from you.

Best regards
Frederick Thomssen
